Tasks/AzureTestPlanV0/ciEventLogger.ts

```diff
@@ -0,0 +1,33 @@
+import * as tl from 'azure-pipelines-task-lib/task';
+
+const area: string = 'TestManagement';
+const feature: string = 'AzureTestPlanTask';
+
+function getDefaultProps() {
+    return {
+        releaseuri: tl.getVariable('Release.ReleaseUri'),
```

Comment:
can we check if it is release / build flow and return only those params?

Reply:
If it is build only scenario, releaseuri and releaseid would be null and wouldn't be populated in ciData.